tools/mcp/daemon/zzz_od_daemon.py

```
@@ -0,0 +1,224 @@
+        return f"[WARN] 端口 {port} 已被占用，可能有其他程序在使用"
+
+    try:
+        # 输出重定向到日志文件:长驻 server 若用 PIPE 且不持续消费,buffer 满会阻塞子进程
+        log_path = PROJECT_ROOT / '.debug' / 'zzz_od_mcp' / 'main_server.log'
+        log_path.parent.mkdir(parents=True, exist_ok=True)
+        log_file = open(log_path, 'w', encoding='utf-8')  # noqa: SIM115 daemon 持有,生命周期内不 close
+        cmd = [
+            'uv', 'run', '--env-file', '.env',
+            'python', '-m', 'zzz_od.backend.entry.server', '--port', str(port),
+        ]
+        process = subprocess.Popen(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)